src/storage/resourceLoader.ts

```diff
@@ -106,6 +132,15 @@ export abstract class ResourceLoader {
     forceDeepRefresh?: boolean,
   ): Promise<Schema[]>;
 
+  /**
+   * Get the schemas associated with a given Kafka topic's cluster's schema registry.
+   * This returns _all_ of the schemas in the registry, not just those possibly associated with the topic.
+   **/
+  public abstract getSchemasForTopicRegistry(
```

Comment:
Nit: the naming is a little weird here; I could see maybe something like `getSchemasForTopicEnvironment` since it's kind of navigating that `topic <- Kafka cluster <- env -> SR -> schemas[]` hill. 🤷

Reply:
That's a fine name. Is going to be clunky regardless 'cause is returning _all_ of the schemas for the registry in the right schema registry. Is just that sometimes we only have a topic onhand when we want them.

Will spitball with you later about how we can optimize the topic viewprovider getChildren() to not re-fetch schemas for each time a topic is expanded.

Reply:
Renamed.